paasta_tools/cli/schemas/marathon_schema.json

```diff
@@ -214,6 +219,10 @@
                 "container_port": {
                     "type": "number"
                 },
+                "containerizer_type": {
```

Comment:
nit: I think 'containerizer' is enough

Reply:
yeah, it worked, look for dsc-test-service.gpu in pnw-devc. I am using float here hoping gpus can be shared in the near future, i.e. a task can have 0.5 gpus.